crates/jstzd/tests/utils.rs

```diff
@@ -39,7 +47,34 @@ pub async fn setup() -> (OctezNode, OctezClient, octez_baker::OctezBaker) {
     activate_alpha(&octez_client).await;
 
     let baker = spawn_baker(&octez_node, &octez_client).await;
-    (octez_node, octez_client, baker)
+    let rollup = spawn_rollup(&octez_node, &octez_client).await;
```

Comment:
Adding this here means that all existing test cases calling this method are going to spin up a rollup node. I don't think this is what we want. Better to call `spawn_rollup` in `crates/jstzd/tests/octez_rollup_test.rs` instead.

Reply:
done